appveyor.yml

```diff
@@ -53,6 +53,7 @@ install:
     - cmd: conda config --add channels http://conda.binstar.org/conda-forge
     - cmd: conda info
     - cmd: conda install -n root --quiet --yes conda-build anaconda-client jinja2 setuptools
+    - cmd: if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "34" conda install conda-build=1.20.0 --yes
```

Comment:
Actually that is not true, @ocefpaf. I only fixed Travis CI and for a different issue. This was my best guess while working on my cell in the airport waiting area moments before boarding and haven't had a chance to look until now. We need to add this in a new PR to `conda-smithy` with @melund's change.